usbsdmux/usbsdmux.py

```diff
@@ -18,12 +18,56 @@
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 
+import os
 import time
 
-from .pca9536 import Pca9536
+from .i2c_gpio import Pca9536, Tca6408
 
 
-class UsbSdMux(object):
+class UsbSdMuxFactory:
```

Comment:
Use a normal function instead of a class which is never instantiated and only has one static method.

Reply:
Good point. Changed